ui/uihandler.go

```diff
@@ -129,47 +139,41 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 		}
 		w.Header().Set("Content-Type", "text/html; charset=UTF-8")
 		err := mainTemplate.Execute(w, &struct {
```

Comment:
remove the : now that err is declared already above

Reply:
done!